packages/components/select/tests/multi-select.test.tsx

```diff
@@ -1,6 +1,23 @@
 import { a11y, render, screen } from "@yamada-ui/test"
 import { Select, Option, MultiSelect } from "../src"
 
+const setupMultiSelect = async (props = {}) => {
+  const { user } = render(
+    <MultiSelect {...props}>
+      <Option value="One">One</Option>
+      <Option value="Two">Two</Option>
+      <Option value="Three">Three</Option>
+    </MultiSelect>,
+  )
+
+  const input = screen.getByRole("combobox")
+  await user.click(input)
+
+  return { user, input }
+}
+
+const getOptions = () => screen.findAllByRole("option")
+
```

Comment:
It is difficult to understand, so please do the same as any other test.
Render the component in `test`.

Reply:
I understand!
I'll fix it!